--- a/third_party/cpuinfo/BUILD.gn
+++ b/third_party/cpuinfo/BUILD.gn
@@ -123,6 +123,12 @@
     ]
   }
 
+  if (current_cpu == "riscv64") {
+    sources = [
+      "src/src/riscv/uarch.c"
+    ]
+  }
+
   # Because "init.c" is reused again, but only has sources in the corresponding
   # build config.
   deps = [ ":x86_cache" ]
@@ -190,6 +196,15 @@
     ]
   }
 
+  if ((is_linux || is_chromeos) && current_cpu == "riscv64") {
+    sources = [
+      "src/src/riscv/linux/api.h",
+      "src/src/riscv/linux/init.c",
+      "src/src/riscv/linux/riscv-hw.c",
+      "src/src/riscv/linux/riscv-isa.c",
+    ]
+  }
+
   if (is_android && (current_cpu == "arm" || current_cpu == "arm64")) {
     sources = [
       "src/src/arm/android/properties.c",
@@ -216,6 +231,15 @@
     ]
   }
 
+  if (is_android && current_cpu == "riscv64") {
+    sources = [
+      "src/src/riscv/linux/api.h",
+      "src/src/riscv/linux/init.c",
+      "src/src/riscv/linux/riscv-hw.c",
+      "src/src/riscv/linux/riscv-isa.c",
+    ]
+  }
+
   if (is_win && (current_cpu == "x86" || current_cpu == "x64")) {
     sources = [ "src/src/x86/windows/init.c" ]
   }
